docs/content/community.md

```diff
@@ -30,6 +30,10 @@ Questions? Problems? Searching for a quick response from the team behind hawkBit
 ## Mailing List
 A great way to stay up to date with hawkBit activity is to subscribe to the Mailing list provided by Eclipse. Sign up for the mailing list [here](https://dev.eclipse.org/mailman/listinfo/hawkbit-dev).
 
+## Stack Overflow
+
+Find existing questions or raise a new one tagged with `eclipse-hawkbit` [here](https://stackoverflow.com/questions/tagged/eclipse-hawkbit).
```

Comment:
```suggestion
We want to use Stack Overflow for questions regarding hawkBit. So if you have any, please have a look [here](https://stackoverflow.com/questions/tagged/eclipse-hawkbit). If you can't find it - raise it and tag it with `eclipse-hawkbit`. Hence, others will benefit from your question, too.
```

Reply:
```suggestion
Visit [stackoverflow.com](https://stackoverflow.com/questions/tagged/eclipse-hawkbit) to find and raise questions tagged with `eclipse-hawkbit`.
```

Reply:
@laverman @StefanKlt thanks for your feedback. What about this one?